src/pages/ContactPage.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="contact-page">
+    <BreadCrumbs :steps="steps" :back-route="{ name: 'Home' }" />
+    <SectionTitle class="contact-page__title">Contact Us</SectionTitle>
+    <div class="contact-page__card">
+      <div class="contact-page__image">
+        <MapImage />
+      </div>
+      <ContactCard class="contact-page__details" />
+      <div class="contact-page__icons">
+        <TweeterIcon class="contact-page__icon" />
+        <FacebookIcon class="contact-page__icon" />
+        <LinkedinIcon class="contact-page__icon" />
+        <InstagramIcon class="contact-page__icon" />
+        <GitIcon class="contact-page__icon" />
+        <BeIcon class="contact-page__icon" />
+        <BasketballIcon class="contact-page__icon" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { reactive, toRefs } from "vue";
+import BreadCrumbs from "../components/common/BreadCrumbs.vue";
+import SectionTitle from "../components/common/SectionTitle.vue";
+import MapImage from "@assets/images/map.svg?component";
+import ContactCard from "../components/common/ContactCard.vue";
```

Comment:
You're using relative imports here, while 
```suggestion
import BreadCrumbs from "@components/common/BreadCrumbs.vue";
import SectionTitle from "@components/common/SectionTitle.vue";
import MapImage from "@assets/images/map.svg?component";
import ContactCard from "@components/common/ContactCard.vue";
```